tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICES file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.eclipse.microprofile.health.tck;
+
+import org.eclipse.microprofile.health.tck.deployment.FailedCheck;
+import org.eclipse.microprofile.health.tck.deployment.SuccessfulCheck;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+import javax.json.JsonValue;
+import java.io.StringReader;
+
+/**
+ * @author Heiko Braun
+ */
+@RunWith(Arquillian.class)
+public class MultipleProceduresFailedTest extends SimpleHttp {
+
+    @Deployment
+    public static Archive getDeployment() throws Exception {
+        WebArchive deployment = ShrinkWrap.create(WebArchive.class, "tck.war");
+        deployment.addClass(FailedCheck.class);
+        deployment.addClass(SuccessfulCheck.class);
+        return deployment;
+    }
+
+    /**
+     * Verifies the health integration with CDI at the scope of a server runtime
+     */
+    @Test
+    @RunAsClient
+    public void testFailureResponsePayload() throws Exception {
+        Response response = getUrlContents("http://localhost:8080/health");
```

Comment:
I think a) is useful @heiko-braun ! For b), what should be happening when more than one apps running and listening on the same port?